src/style.css

```diff
@@ -1,3 +1,92 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  /* width: 100%; */
+  margin-bottom: 20px;
+  width: 1024px;
+}
+
+.logo_png {
+  display: block;
+  width: 40px;
+  height: 40px;
+
+  /* position: relative; */
+
+  /* top: 10px; */
+}
+
+a.blue {
+  color: var(--main-color);
+}
+
+.container {
+  display: flex;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 20px;
+  margin-top: 0;
+  list-style: none;
+  justify-content: center;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  justify-content: center;
+  margin-right: 20px;
+  padding-bottom: 23px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  padding: 22px 0 0;
+  font-size: 12px;
+  height: 15px;
+  color: #000;
+  justify-content: center;
+  align-items: center;
+
+  /* height: 100%; */
+
+  /* display: flex; */
+}
+
+.nav__list :hover {
+  color: var(--main-color);
+}
+
+a.is-active::after {
```

Comment:
The `a.is-active::after` selector is used to style active links. Ensure that this class is applied correctly in the HTML to the intended elements to achieve the desired effect.